hydro/distribution.yaml

```diff
@@ -9270,5 +9270,20 @@ repositories:
       url: https://github.com/stonier/zeroconf_msgs.git
       version: hydro-devel
     status: developed
+  IIWA:
+    doc:
+      type: git
+      url: https://khansari@bitbucket.org/khansari/iiwa.git
+      version: hydro-devel
+    release:
+      tags:
+        release: release/hydro/{package}/{version}
+      url: https://bitbucket.org/khansari/iiwa/wiki/Home
```

Comment:
This url must point to a git-buildpackage release repository. Please read about `bloom` and how to release a ROS package.